bliss/simulator/prior.py

```diff
@@ -249,6 +272,45 @@ def _draw_truncated_pareto(alpha, min_x, max_x, n_samples) -> Tensor:
         uniform_samples = torch.rand(n_samples) * u_max
         return min_x / (1.0 - uniform_samples) ** (1 / alpha)
 
+    def _compute_flux_ratio(self, obj_fluxes, run, camcol, field, field_dir, gains):
```

Comment:
Maybe this is fine for this PR since everything is still in terms of SDSS, but we should eventually move this to the SDSS object itself